pancancer_utilities/data_utilities.py

```diff
@@ -5,6 +5,7 @@
 import os
```

Comment:
A very minor suggestions: according to PEP 8 E302, two blank lines should be left between methods.